README.md

```diff
@@ -242,36 +184,21 @@ NSQLOOKUPD_TCP_ADDRESS='127.0.0.1:4160'
 NSQLOOKUPD_HTTP_ADDRESS='127.0.0.1:4161'
 ```
 
-See the [`.sample.env`](examples/.sample.env) file
-for more detail.
+See the [`.sample.env`](examples/.sample.env) file for more detail.
 
 ### Testing Against the Fake
+In the gem's test suite, the fake message queue is used.
 
-In the gem's test suite,
-the fake message queue is used.
-
-If you would like to force
-use of the real NSQ adapter,
-ensure `FAKE_QUEUE` is set to `false`.
+If you would like to force use of the real NSQ adapter, ensure `FAKE_QUEUE` is set to `false`.
 
-When you are developing your application,
-it is recommended to
-start by using the fake queue:
+When you are developing your application, it is recommended to start by using the fake queue:
 
 ```shell
 FAKE_QUEUE=true
 ```
+Also be sure call `FakeMessageQueue.reset!` before each test in your app to ensure there are no leftover messages.
 
-Also be sure call
-`FakeMessageQueue.reset!`
-before each test in your app to ensure
-there are no leftover messages.
-
-Also note that during gem tests,
-we are aliasing `MessageProcessor` to `SampleMessageProcessor`.
-You can also refer to the latter
-as an example of how
-you might write your own processor.
+Also note that during gem tests, we are aliasing `MessageProcessor` to `SampleMessageProcessor`. You can also refer to the latter as an example of how you might write your own processor.
```

Comment:
IMHO: arguing about line length and linefeeds is the worst kind of bike shedding.